server/archived_users/methods.js

```diff
@@ -1,69 +1,87 @@
 Meteor.methods({
-  removeUser:function(userId, incident){
+  removeUser: function(userId, incident) {
     check(userId, String);
     check(incident, {
       action: String,
       matter: String,
       detail: String
     });
-    const user = Meteor.users.findOne({_id:userId});
-    const accessToken = user.services.slack.accessToken;
 
     if (this.userId !== userId) {
-      throw new Meteor.Error('Users.methods.removeUser.not-a-currentuser-in', 'Must be logged in to Delete an account.');
+      throw new Meteor.Error(
+        "Users.methods.removeUser.not-a-currentuser-in",
+        "Must be logged in to Delete an account."
+      );
     }
 
-    //removing access for codebuddies slack app
-    const result = authRevoke(accessToken);
-    if(result.ok){
+    const user = Meteor.users.findOne({ _id: userId });
+    let accessToken = null;
+    let result = null;
+    if (user.services.slack && user.services.slack.accessToken) {
```

Comment:
When user has used both services for sign in (slack & github), user would have both of them available. In that case  system will only revoke the slack token and not revoke the github one. 
Here, we should check for both services separately and revoke them accordingly.  

Very roughly :
```
if (slackTokenExists) {
 // remove slack auth
}
if (githubTokenExists) {
 // remove github auth
}
```

Reply:
Will fix that later

Reply:
Will fix that later

Reply:
Will fix it later.